File: JavaScriptFrontEnd/Syntax Fundamentals/Exercise/vacation.js
function vacation(countNumber, strType, strDay){

    let count = Number(countNumber);
    let type = String(strType);
    let day = String(strDay);

    let total = 0;

    switch(day){
        case 'Friday' :
            switch(type){
                case 'Students' :
                   total=8.45*count;
                   if(count>=30){
                    total=total*0.85;
                   }
                    break;
                case 'Business' :
                    total=10.90*count;
                    if(count>=100){
                        total=total-109;
                    }
                    break;
                case 'Regular' :
                    total=15*count;
                    if(count>=10 && count<=20){
                        total=total*0.95;
                    }
                    break;
            }break;
        
        case 'Saturday' :
            switch(type){
                case 'Students' :
                    total=9.80*count;
                    if(count>=30){
                        total=total*0.85;
                       }
                    break;
                case 'Business' :
                    total=15.60*count;
                    if(count>=100){
                        total=total-156;
                    }
                    break;
                case 'Regular' :
                    total=20*count;
                    if(count>=10 && count<=20){
                        total=total*0.95;
                    }
                    break;
            }break;

        case 'Sunday' :
            switch(type){
                case 'Students' :
                    total=10.46*count;
                    if(count>=30){
                        total=total*0.85;
                       }
                    break;
                case 'Business' :
                    total=16*count;
                    if(count>=100){
                        total=total-160;
                    }
                    break;
                case 'Regular' :
                    total=22.50*count;
                    if(count>=10 && count<=20){
                        total=total*0.95;
                    }
                    break;
            }break;
    }
    console.log('Total price: '+total.toFixed(2))
}

File: JavaFundamentals/BasicSyntaxConditionalStatements/Exercise/Vacation.java
package FundamentalsModule.BasicSyntaxConditionalStatements.Exercise;

import java.util.Scanner;
public class Vacation {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double input = Double.parseDouble(scanner.nextLine());
        String first = scanner.nextLine();
        String second = scanner.nextLine();
        double result = 0;


        if(first.equals("Students")){if(second.equals("Friday")){result=8.45*input;}}
        if(first.equals("Students")){if(second.equals("Saturday")){result=9.80*input;}}
        if(first.equals("Students")){if(second.equals("Sunday")){result=(10.46*input);}}
        if(first.equals("Students")){if(input>=30){result=result*0.85;}}

        if(first.equals("Business")){if(second.equals("Friday")){ if(input>=100){input=input-10;}result=10.90*input;}}
        if(first.equals("Business")){if(second.equals("Saturday")){ if(input>=100){input=input-10;}result=15.60*input;}}
        if(first.equals("Business")){if(second.equals("Sunday")){ if(input>=100){input=input-10;}result=16*input;}}

        if(first.equals("Regular")){if(second.equals("Friday")){result=(15*input);}}
        if(first.equals("Regular")){if(second.equals("Saturday")){result=20*input;}}
        if(first.equals("Regular")){if(second.equals("Sunday")){result=(22.50*input);}}
        if(first.equals("Regular")){if(input>=10 && input<=20){result=result*0.95;}}


        System.out.printf("Total price: %.2f", result);


    }
}
